xml/System.Transactions/TransactionInterop.xml

```diff
@@ -97,7 +97,7 @@
       <Docs>
         <param name="transaction">A <see cref="T:System.Transactions.Transaction" /> instance to be marshaled.</param>
         <summary>Gets an <see cref="T:System.Transactions.IDtcTransaction" /> instance that represents a <see cref="T:System.Transactions.Transaction" />.</summary>
-        <returns>An <see cref="T:System.Transactions.IDtcTransaction" /> instance that represents a <see cref="T:System.Transactions.Transaction" />.  The <see cref="T:System.Transactions.IDtcTransaction" /> instance is compatible with the unmanaged form of [ITransaction](http://msdn.microsoft.com/library/default.asp?url=/library/cossdk/html/d40264be-f9b2-4044-aac6-84b43e5be666.asp) used by MSDTC and with the Managed form of <see cref="T:System.EnterpriseServices.ITransaction" /> used by <see cref="N:System.EnterpriseServices" />.</returns>
```

Comment:
This link is dead, but I couldn't find a suitable replacement. Any suggestions?

Reply:
<https://docs.microsoft.com/en-us/previous-versions/windows/desktop/ms686531(v=vs.85)>